tools/genesis_generator/.gitignore

```diff
@@ -0,0 +1 @@
+genesis*.json
```

Comment:
They should not be ignored

Reply:
These are results of the tool execution (generated genesis json) this folder isn't a place for these files imo

Reply:
But why do you place files in there? The correct folder is "generated" and they should be gitted